pages/style-guide/html.md

```diff
@@ -4,64 +4,110 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
 
-```
-<!doctype html>
-```
+The markup should be valid HTML5 and it must pass the automated testing we use via grunt-html.
```

Comment:
Yes, we use [grunt-html](https://www.npmjs.com/package/grunt-html) which has an `htmllint` task.